doc/Userland.md

```diff
@@ -272,6 +272,13 @@ Applications can specify their required stack and heap sizes by defining the
 make variables `STACK_SIZE` and `APP_HEAP_SIZE`, which default to 2K and 1K
 respectively as of this writing.
 
+`libtock` will set the stack pointer during startup. To allow each application
+to set its own stack size, `libtock` expects an external symbol `STACK_SIZE` to
```

Comment:
This is no longer true, correct? The value is in the crt0 header, and the userland app shouldn't expect to find `STACK_SIZE` defined.

Reply:
This is wrong, I'll update it now. The value is not in the crt0 header (there isn't one anyone, that was a failed experiment), it's defined during build and placed in the application header by the linker script.